src/api/app/models/event/request.rb

```diff
@@ -81,13 +81,12 @@ def payload_with_diff
 
       ret = payload
       payload['actions'].each do |a|
-        diff = calculate_diff(a)
+        diff = calculate_diff(a).try(:lines)
         next unless diff
-        diff = diff.lines
-        dl = diff.length
-        if dl > DIFF_LIMIT
+        diff_length = diff.length
```

Comment:
[nitpick] I would prefer to inline the diff.length, it is only used two times and IIRC length is already stored as an attribute internally.